src/Microsoft.AspNet.Mvc.Core/ConsumesAttribute.cs

```diff
@@ -184,5 +189,15 @@ private List<MediaTypeHeaderValue> GetContentTypes(string firstArg, string[] arg
 
             return contentTypes;
         }
+
+        /// <inheritdoc />
+        public void SetContentTypes(IList<MediaTypeHeaderValue> contentTypes)
+        {
+            contentTypes.Clear();
```

Comment:
Is this the right behavior? Won't this mean that 
```C#
[Consumes("application/json")]
[Consumes("application/xml")]
```
will result in ```[Consumes("application/json")]``` being overriden/ignored by ```[Consumes("application/xml")]```?